src/js/jsx/sections/export/ExportList.jsx

```diff
@@ -226,28 +226,29 @@ define(function (require, exports, module) {
 
                 return (
                     <div className="layer-exports__header" >
-                        <div className="formline">
+                        <div className="formline formline__space-between">
                             <Label
                                 title={strings.EXPORT.TITLE_SCALE}
                                 size="column-4"
                                 className="label__medium__left-aligned">
                                 {strings.EXPORT.TITLE_SCALE}
                             </Label>
-                            <Gutter />
                             <Label
                                 title={strings.EXPORT.TITLE_SUFFIX}
                                 size="column-6"
                                 className="label__medium__left-aligned">
                                 {strings.EXPORT.TITLE_SUFFIX}
                             </Label>
-                            <Gutter />
                             <Label
                                 title={strings.EXPORT.TITLE_SETTINGS}
                                 size="column-4"
                                 className="label__medium__left-aligned">
                                 {strings.EXPORT.TITLE_SETTINGS}
                             </Label>
-                            <Gutter />
+                            <Button
+                                className="layer-exports__delete-button">
+                                <SVGIcon CSSID="dummy" />
+                            </Button>
```

Comment:
So instead of actually making a dummy icon, you can just make a div with class column-xx (whatever size is needed) to fill the horizontal space. You should be able to get rid of the dummy SVG altogether, and not use the Button component here (using components that are at all complex has a penalty)